docs/_index.yaml

```diff
@@ -27,18 +27,18 @@ landing_page:
         classname: button
   - classname: devsite-landing-row-cards
     items:
+    - heading: "TensorBoard.dev: a new way to share your ML experiment results"
+      image_path: /resources/images/tf-logo-card-16x9.png
+      path: https://blog.tensorflow.org/2019/12/introducing-tensorboarddev-new-way-to.html
+      buttons:
+      - label: "Read on TensorFlow blog"
+        path: https://blog.tensorflow.org/2019/12/introducing-tensorboarddev-new-way-to.html
     - heading: "The What-If Tool: Code-Free Probing of Machine Learning Models"
       image_path: /resources/images/tf-logo-card-16x9.png
       path: https://ai.googleblog.com/2018/09/the-what-if-tool-code-free-probing-of.html
       buttons:
       - label: "Read on Google AI blog"
         path: https://ai.googleblog.com/2018/09/the-what-if-tool-code-free-probing-of.html
-    - heading: "Interactive supervision with TensorBoard"
```

Comment:
Do we actually need to remove something to make space? While this link is old, it might still be relevant.

Reply:
Rows can contain 2-4 cards---but recommend 3 or 4 (since 2 looks odd).
After 4 you need to create a new row.

Reply:
I feel like more than 4 would look crowded, at least on my setup.

![image](https://user-images.githubusercontent.com/547150/70174330-ae1ef980-16a2-11ea-842d-0008f69fcc21.png)

Reply:
Agree. I prefer 3 across

Reply:
Ok, so how about the following:  We remove the links to What-If tool and GitHub, and add a link to the Dev Summit talk?

The remaining three cards will be :
1. TensorBoard.dev blog post
2. TensorBoard.dev Dev Summit YouTube video.
3. Hands-on TensorBoard YouTube video

Users who wish to navigate to the TensorBoard github page may be able to by clicking the "GitHub" link at the top, which includes all of the projects under TensorFlow, including TensorBoard.

Reply:
Got it, thanks @lamberta! 

I agree with the suggested three cards (assuming 2nd card is just TensorBoard dev summit talk).